2/README.md

```diff
@@ -0,0 +1,122 @@
+
+### Draft Specifications
+When raw specifications can be demonstrated, they become **draft** specifications. Changes to draft specifications should be done in consultation with users. Draft specifications are contracts between the editors and implementers.
+
+### Stable Specifications
+When draft specifications are used by third parties, they become **stable** specifications. Changes to stable specifications should be restricted to cosmetic ones, errata and clarifications. Stable specifications are contracts between editors, implementers, and end-users.
+
+### Deprecated Specifications
+When stable specifications are replaced by newer draft specifications, they become **deprecated** specifications. Deprecated specifications should not be changed except to indicate their replacements, if any. Deprecated specifications are contracts between editors, implementers and end-users.
+
+### Retired Specifications
+When deprecated specifications are no longer used in products, they become **retired** specifications. Retired specifications are part of the historical record. They should not be changed except to indicate their replacements, if any. Retired specifications have no contractual weight.
+
+### Deleted Specifications
+Deleted specifications are those that have not reached maturity (stable) and were discarded. They should not be used and are only kept for their historical
+value. Only Raw and Draft specifications can be deleted.
+
+## Editorial control
+A specification MUST have a single responsible editor, the only person
+who SHALL change the status of the specification through the lifecycle stages.
+
+A specification MAY also have additional contributors who contribute changes to it. It is RECOMMENDED to use the [C4 process](../1/README.md) to maximize the scale and diversity of contributions.
+
+The editor is responsible for accurately maintaining the state of specifications and for handling all comments on the specification.
+
+## Branching and Merging
+Any member of the domain MAY branch a specification at any point. This is done by copying the existing text, and creating a new specification with the same name and content, but a new number. The ability to branch a specification is necessary in these circumstances:
+
+* To change the responsible editor for a specification, with or without the cooperation of the current responsible editor.
+* To rejuvenate a specification that is stable but needs functional changes. This is the proper way to make a new version of a specification that is in stable or deprecated status.
+* To resolve disputes between different technical opinions.
+
+The responsible editor of a branched specification is the person who makes the branch.
+
+Branches, including added contributions, are derived works and thus licensed under the same terms as the original specification. This means that contributors are guaranteed the right to merge changes made in branches back into their original specifications.
```

Comment:
If we decide to ask all contributors to dedicate all specs to the public domain (e.g. using CC0), then there is no license. This paragraph would have to be reworded, e.g.

Branches, including added contributions, SHOULD be dedicated to the public domain using CC0 (just like the original specification). This means that contributors are guaranteed the right to merge changes made in branches back into their original specifications.